_includes/sites-world-map.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://openlayers.org/en/v5.3.0/css/ol.css" type="text/css">
+  <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=requestAnimationFrame,Element.prototype.classList,URL"></script>
+
+  <style>
+    .map {
+      height: 100%;
+      width: 100%;
+    }
+  </style>
+
+  <!-- Grab whole OpenLayers package from source -->
+  <script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js"></script>
+  <title>Cylc Usage Across the World</title>
+</head>
+
+<body>
+
+  <div id="map" class="map"></div>
+  <script type="text/javascript">
+    // Lat-lon decimal degrees coordinates (to ideally 5 s.f.) for major...
+    // ... known Cylc-using sites, with keys as neat names for labelling:
+    var cylcSiteLatLonCoors = {
+      "Met Office": [50.72762, -3.47564],
```

Comment:
I think it would be nice if we could have this list of sites as a variable like

```js
const cylcSites = [
{
name: "Met Office",
latlon: [50.72762, -3.47564],
continent: 'Europe'
},
...
]
```

or organize by continents if easier. And then derive from the list both the map, as well as the list below. So maintaining it would be easier, as we could update just one place in a file.

Reply:
Good idea! For extensions, I was thinking we could display extra information via e.g. the node colours, for example to distinguish sites where Cylc is developed as well as used i.e. MO, NIWA (& BoM?), or UM partner sites, & this as a data structure could make that much easier to add that in, by setting a relevant key to a Boolean & processing that.

When I get around to revising this, I will get this data structure set up.